python.d/neo.chart.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# Description:  netdata python.d module for monitoring Neo4j
+# Author: Jerome Baton, copied from the example of Pawel Krupa (paulfantom)
+
+
+# if you want to add a chart:
+#     insert it in ORDER
+#     copy paste and rename a chart in CHARTS
+#     code your query to return two results per line (label , value)
```

Comment:
Long term, it probably makes sense to change this so that the user doesn't have to edit the plugin itself, but can just modify the configuration file to get additional dimensions.